src/app/ngrx/reducers/index.ts

```diff
@@ -76,6 +76,18 @@ export const selectEpisodesError = createSelector(selectEpisodeState, getEpisode
 export const selectSelectedPageEpisodes = createSelector(selectEpisodes, selectPageRoute, (episodes: EpisodeModel[], page: number) => {
   return episodes.filter(episode => episode.page === page);
 });
+export const selectMostRecentEpisode = createSelector(selectSelectedPodcast, selectEpisodes, (pod: PodcastModel, eps: EpisodeModel[]) => {
+  // TODO: if you didn't start on the first page, this isn't the most recent
+  let recent: EpisodeModel;
+  eps.forEach(episode => {
+    if (episode.seriesId === pod.seriesId) {
```

Comment:
I'm not entirely clear on why i have to do this.  The above `selectSelectedPageEpisodes` uses the same `selectEpisodes` ... but it doesn't have to filter by seriesId.  What the what?